docs/_posts/kolia1985/2021-12-09-text_detection_v1_en.md

```diff
@@ -25,23 +25,91 @@ CRAFT: Character-Region Awareness For Text detection, is designed with a convolu
 
 {:.btn-box}
 <button class="button button-orange" disabled>Live Demo</button>
-<button class="button button-orange" disabled>Open in Colab</button>
+[Open in Colab](https://colab.research.google.com/github/JohnSnowLabs/spark-ocr-workshop/blob/master/tutorials/Certification_Trainings/others/SparkOcrImageTextDetection.ipynb){:.button.button-orange.button-orange-trans.co.button-icon}
 [Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/clinical/ocr/text_detection_v1_en_3.0.0_3.0_1639033905025.zip){:.button.button-orange.button-orange-trans.arr.button-icon}
 
 ## How to use
 
 
-
 <div class="tabs-box" markdown="1">
 {% include programmingLanguageSelectScalaPythonNLU.html %}
 ```python
-text_detector = ImageTextDetector.pretrained("text_detection_v", "en", "clinical/ocr")
-text_detector.setInputCol("image")
-text_detector.setOutputCol("text_regions")
+
+    from pyspark.ml import PipelineModel
+    from sparkocr.transformers import *
+    
+    imagePath = "path to image"
+    image_df = spark.read.format("binaryFile").load(imagePath)
+
+    binary_to_image = BinaryToImage() 
+    binary_to_image.setImageType(ImageType.TYPE_3BYTE_BGR)
+    
+    text_detector = ImageTextDetectorV2 \
```

Comment:
model in name and download parts --> **text_detection_v1**

but model name in the pipeline  --> **image_text_detector_v2**.

Reply:
solved